README.md

```diff
@@ -286,6 +120,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] It would be better if we create the MIT license file in the root directory. Currently, there is no MIT license file in the project